data/locale/meson.build

```diff
@@ -1,5 +1,5 @@
 if get_option('gui').enabled()
-    languages = ['de']
+    languages = ['de fr']
```

Comment:
This should be:

    languages = ['de', 'fr']

i.e. comma-separated list of strings inside the square brackets:

https://mesonbuild.com/Syntax.html#arrays

Reply:
fixed now, thanks